### ISSUE TITLE
z-w and z-i seem to do the opposite of what I'd expect

### ISSUE THREAD

**Issue by max-sixty** [kahole/vscode-magit#218]
Thanks for the awesome library! It's always in my list of recommendations to others, so thank you.

Not sure if this is just me but:
- "Save worktree" stashes the index
- "Save index" stashes the worktree

I would have expected the opposite, is that correct?

Edit: this seems to have been raised in https://github.com/kahole/edamagit/issues/208 too

**Comment by kahole** [comment]
TL;DR: Swapped them and changed the verb from "Save" to "Stash"
.....

I actually swapped them back in april, if you see the commit referencing #208. They are actually correct to the way emacs magit was up until recently (don't know exactly when).
"Save" = "Save from being stashed", which is what the author of #208 was pointing out.

This all has to do with "Save x" vs "Stash x". Emacs magit used to have the same verbs as edamagit, namely "Save x".

Seems like this description was/is confusing some users, evident in edamagit issue history also, so they changed it.
And I just did too! I swapped them back and changed the description from "Save x" to "Stash x". So now everything should match today's emacs magit.

**Comment by max-sixty** [comment]
Awesome, thanks @kahole !

Yeah, I think some of the confusing is that `git stash` is short for `git stash push / save`, so I thought that's where the `save` reference was from.

FYI I still see "save keeping index", if you intended to change that. It's really confusing for me though.
